packages/@aws-cdk/aws-ec2/lib/machine-image.ts

```diff
@@ -374,15 +381,24 @@ export class AmazonLinuxImage extends GenericSSMParameterImage {
    * Return the SSM parameter name that will contain the Amazon Linux image with the given attributes
    */
   public static ssmParameterName(props: AmazonLinuxImageProps = {}) {
-    const generation = (props && props.generation) || AmazonLinuxGeneration.AMAZON_LINUX;
-    const edition = (props && props.edition) || AmazonLinuxEdition.STANDARD;
-    const virtualization = (props && props.virtualization) || AmazonLinuxVirt.HVM;
-    const storage = (props && props.storage) || AmazonLinuxStorage.GENERAL_PURPOSE;
-    const cpu = (props && props.cpuType) || AmazonLinuxCpuType.X86_64;
+    let generation = (props && props.generation) || AmazonLinuxGeneration.AMAZON_LINUX;
```

Comment:
I'd prefer we keep the `const`s where possible.
```suggestion
    const generation = (props && props.generation) || AmazonLinuxGeneration.AMAZON_LINUX;
```